config.toml.dist

```diff
@@ -1,28 +1,59 @@
+[server]
+# IP address to bind to - Default: "127.0.0.1"
 listen_ip = "127.0.0.1"
+
```

Comment:
I think this'd be much more readable if we just have a newline between sections, not every field.